src/api.ts

```diff
@@ -66,12 +68,18 @@ export {
   typecheck,
   walk,
 } from "mobx-state-tree";
+export { action, ClassModel, register, view } from "./class-model";
```

Comment:
It doesn't look like any of the are used. We should probably also add `prettier-plugin-organize-imports` to the prettier config here.

Reply:
This is an `export` not an `import`, thats ok right?

Reply:
Oh oops yep sorry!